dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/task/AbstractTask.java

```diff
@@ -106,4 +106,23 @@ public T run(TaskRunContext context) throws Exception {
     sink.commit();
     return result;
   }
+
+  protected String describeSourceData() {
+    return "from " + getClass().getSimpleName();
+  }
+
+  /**
+   * Returns the source data description for the SQL query.
+   *
+   * @param query SQL query
+   * @return the source data description
+   */
+  protected static String createSourceDataDescriptionForQuery(String query) {
+    return "from\n        " + query;
```

Comment:
nit: how about something similar to "from\n SQL query: " ? It's hard to count spaces.

Reply:
I have preserved the spaces to maintain the current pretty-printed output.

```
********************************************************************
* Task Summary:
* Task SUCCEEDED (INFORMATIONAL) Write compilerworks-version.txt from product version information.
* Task SUCCEEDED (OPTIONAL) Write dbc.ResUsageSpma_2023-03-28T10:00:00Z.csv from
        SELECT * FROM dbc.ResUsageSpma WHERE TheTimestamp >= 1679997600 AND TheTimestamp < 1680001200
* Task SUCCEEDED (REQUIRED) Write utility_logs_2023-03-28T10:00:00Z.csv from
        SELECT AcctString, AcctStringDate, ...
* Task SUCCEEDED (REQUIRED) Write compilerworks-metadata.yaml containing dump metadata.
* Task SUCCEEDED (REQUIRED) Write compilerworks-format.txt containing 'teradata.logs.zip'.
* Task SUCCEEDED (OPTIONAL) Write dbc.ResUsageScpu_2023-03-28T10:00:00Z.csv from
        SELECT * FROM dbc.ResUsageScpu WHERE TheTimestamp >= 1679997600 AND TheTimestamp < 1680001200
* Task SUCCEEDED (INFORMATIONAL) Write compilerworks-arguments.txt from command-line arguments.
* Task SUCCEEDED (REQUIRED) Write query_history_2023-03-28T10:00:00Z.csv from
        SELECT L.QueryID, ST.SQLRowNo, ST.SQLTextInfo, ...
```